remote-sr/src/main/resources/META-INF/remotesr.config.properties

```diff
@@ -1,3 +1 @@
-chrome=
-chrome_default_mac=/Applications/Google Chrome.app/Contents/MacOS/Google Chrome
-chrome_default_windows=/Program Files (x86)/Google/Chrome/Application/chrome.exe
```

Comment:
Why remove sensible defaults? This is going to cause a lot of unnecessary pain for everyone in the team because they will have to configure the system manually. I don't think this is reasonable